src/base/settingsstorage.cpp

```diff
@@ -30,6 +30,7 @@
 #include "settingsstorage.h"
 
 #include <memory>
+#include <QDebug>
```

Comment:
I forgot to remove it after some testing.
By the way, if you're interested, I found one unpleasant thing in qsettings (apparently it refers to INI format).